packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.base.tsx

```diff
@@ -161,6 +161,21 @@ export class HoverCardBase extends BaseComponent<IHoverCardProps, IHoverCardStat
     }
   }
 
+  private _getEventListenerElement(): HTMLElement | undefined {
```

Comment:
Can you please reuse the `_getTargetElement` function somehow? It just feels wrong to have only the `default` case of the switch statement differ in logic between this 2 functions. Thanks!